config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -76,6 +83,68 @@ presubmits:
           --timeout=65m
           "--node-args=--image-config-file=${GOPATH}/src/k8s.io/test-infra/jobs/e2e_node/containerd/containerd-main-presubmit/image-config-presubmit.yaml -node-env=PULL_REFS=$(PULL_REFS)"
 
+  - name: pull-containerd-node-e2e-canary
+    always_run: true
+    max_concurrency: 8
+    decorate: true
+    cluster: k8s-infra-prow-build
+    branches:
+    - main
+    decoration_config:
+      timeout: 100m
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    annotations:
+      testgrid-dashboards: sig-node-containerd,containerd-presubmits
+      testgrid-tab-name: pull-containerd-node-e2e
+      description: run node e2e tests
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - name: pull-containerd-node-e2e
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230719-72ce785784-master
+        env:
+        - name: USE_TEST_INFRA_LOG_DUMPING
+          value: "true"
+        - name: DEPLOY_BUCKET
+          value: k8s-staging-cri-tools
+        command:
+        - sh
+        - -c
+        - >
+          runner.sh
+          ./test/build.sh
+          &&
+          cd ${GOPATH}/src/k8s.io/kubernetes
+          &&
+          kubetest2
+          noop
+          --test=node
+          --
+          --repo-root=.
+          --gcp-zone=us-west1-b
+          --parallelism=8
+          --focus-regex="\[NodeConformance\]\[NodeFeature:.+\]|\[NodeFeature\]"
+          --skip-regex="\[Flaky\]|\[Slow\]|\[Serial\]"
+          --node-env="PULL_REFS=$(PULL_REFS)" # doesn't exist yet in kubetest2 or in k/k
```

Comment:
--node-env doesn't exist in the make target that kubetest2 invokes.

I need to fix it in k/k and k-sigs/kubetest2

Reply:
https://github.com/kubernetes/kubernetes/pull/119513 k/k patch